test/OrchardCore.Commerce.Tests.UI/Tests/OrderTests/BehaviorOrderTests.cs

```diff
@@ -0,0 +1,157 @@
+using Atata;
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OrchardCore.Commerce.Tests.UI.Tests.OrderTests;
+
+public class BehaviorOrderTests : UITestBase
+{
+    public BehaviorOrderTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    [Theory, Chrome]
+    public Task OrderEditorShouldWorkProperly(Browser browser) =>
+        ExecuteTestAfterSetupAsync(
+            async context =>
+            {
+                await context.SignInDirectlyAndGoToDashboardAsync();
+                await context.CreateNewContentItemAsync("Order");
+
+                await AddNewProductAsync(context);
+
+                // Total value should be 0 by default.
+                AssertTotal(context, "$", "0");
+                await context.ClickAndFillInWithRetriesAsync(By.Name("OrderPart.LineItems[0].Quantity"), "5");
```

Comment:
I've noticed that you are using `By.Name` or `By.Id` seemingly randomly. I think all fields have both and you can consistently use `By.Id` everywhere.

Reply:
The Product editor's fields do not have `id`s, only `name`s.
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/51411356/0751759c-3c20-456c-8e72-31ad351fbd70)

There are a few cases where I mistakenly used `By.Id()` in a `Missing()` method when checking for fields without `id`s, I'll fix that.

I can also add `id` attributes to the above mentioned fields, let me know if that's preferable to using the `name` attributes.

Reply:
Right, I was thinking about inputs created with `asp-for` attribute. Then please use `By.Name` everywhere.